packages/manifest/src/pixi.ts

```diff
@@ -172,13 +174,27 @@ function collect(
                 ) as PixiManifestEntry[];
         }
 
-        result.forEach((entry) =>
+        const hasIgnoreFileExtensions
+            = options.ignoreFileExtensions !== undefined
+            && options.ignoreFileExtensions.length > 0;
+
+        result.forEach((entry, index) =>
         {
-            if (
-                !entry.data?.tags
-                && (Object.keys(tree.fileTags).length > 0
-                    || Object.keys(tree.pathTags).length > 0)
-            )
+            if (hasIgnoreFileExtensions)
+            {
+                if (
+                    options.ignoreFileExtensions?.some((extensionName) =>
+                        entry.srcs[0].endsWith(extensionName)
```

Comment:
Q: will there be a case where `srcs` is more than 1?

Reply:
So as discussed I think there will be different sources if you have settings in your assetpack config to generate different files, i.e generating hd or sd assets, generating webp etc.